doc/us-en/FAQ.md

```diff
@@ -64,4 +64,26 @@ export DOCKER_API_VERSION=1.38
 
 **Question 7**: How does BIE access the NB-IOT network?
 
-NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
\ No newline at end of file
+NB-IoT is a network standard similar to 2/3/4G with low bandwidth and low power consumption. NB-IoT supports TCP-based MQTT protocol, so you can use NB-IoT card to connect to Baidu Cloud IotHub, deploy OpenEdge application and communicate with BIE Cloud Management Suite. However, among the three major operators in China, Telecom have imposed whitelist restrictions on their NB cards, and only allow to connect to Telecom Cloud Service IP. Therefore, only Mobile NB cards and Unicom NB cards can be used to connect to Baidu Cloud Service.
+
+**Question 8**: var/run/openedge.sock: address already in use
+
+Remove var/run/openedge.sock and restart Openedge.
+
+**Question 9**: Does openedge support to push data to kafka?
+
+For support, you can use the function framework to write a function that is responsible for subscribing messages from the Hub and writing them to Kafka one by one. You can also customize the module, which subscribes to the Hub and then writes kafka in bulk.
+
+**Question 10**: What are the ways to change openedge configurations? Can I only make configuration changes through the BIE Cloud Management Suite?
+
+Currently, we recommend changing configurations through the BIE Cloud Management Suite, but you can also manually change the configuration file on the core device and then restart openedge to take effect.
+
+**Question 11**: Does the openedge function framework support multiple running instances of function? How to configure?
+
+The function framework would start multiple running instances based on the real-time computing load. The core configuration is as follows:
+```
+instance: function instance configuration
+  min: The default value is `0`, means the minimum number of function instance. And the minimum configuration allowed to be set is `0`, the maximum configuration allowed to be set is `100`.
+  max: The default value is `1`, means the maximum number of function instance. And the minimum configuration allowed to be set is `1`, the maximum configuration allowed to be set is `100`.
+```
+For more function configuration: [Local Function Configuration](https://github.com/baidu/openedge/blob/master/doc/us-en/tutorials/Config-interpretation.md#local-function-configuration)
```

Comment:
For more function configuration: [Local Function Configuration](./tutorials/Config-interpretation.md#local-function-configuration)